examples/llm/src/models/mosaic_gpt.py

```diff
@@ -133,18 +133,47 @@ def _attn_bias(self, device, dtype):
 
     def forward(self,
                 input_ids: torch.LongTensor,
-                key_padding_mask: Optional[torch.ByteTensor] = None):
+                attention_mask: Optional[torch.ByteTensor] = None,
```

Comment:
why change it from `key_padding_mask`?

Reply:
yeah, huggingface expects it to be called `attention_mask`. In principle, we could change its name in `prepare_inputs_for_generation`, but i'd rather just follow their expected forward signature.